easybuild/easyconfigs/g/GROMACS/GROMACS-2021.5-foss-2021b-CUDA-11.4.1-PLUMED-2.8.0.eb

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright:: Copyright 2012-2016 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+# Authors::
+# * Wiktor Jurkowski <wiktor.jurkowski@gmail.com>
+# * Fotis Georgatos <fotis@cern.ch>
+# * George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Adam Huffman <adam.huffman@crick.ac.uk>
+# * Ake Sandgren <ake.sandgren@hpc2n.umu.se>
+# * Dugan Witherick <dugan.witherick@warwick.ac.uk>
+# * Christoph Siegert <christoph.siegert@uni-leipzig.de>
+# * J. Sassmannshausen <j.sassmannshausen@imperial.ac.uk>
+# License::   MIT/GPL
+
+name = 'GROMACS'
+version = '2021.5'
+_plumedver = '2.8.0'
+_cuda_suffix = '-CUDA-%(cudaver)s'
+versionsuffix = _cuda_suffix + '-PLUMED-%s' % _plumedver
+
+homepage = 'https://www.gromacs.org'
+description = """
+GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
+Newtonian equations of motion for systems with hundreds to millions of
+particles.
+
+This is a GPU enabled build, containing both MPI and threadMPI builds
+for both single and double precision.
+
+It also contains the gmxapi extension for the single precision MPI build
+next to PLUMED."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [
+    'https://ftp.gromacs.org/pub/gromacs/',
+    'ftp://ftp.gromacs.org/pub/gromacs/',
+]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    'GROMACS-2019_fix_omp_num_threads_and_google_test_death_style_in_tests.patch',
+    'GROMACS-2019_increase_test_timeout_for_GPU.patch',
+    'GROMACS-2021_fix_gmxapi_gmx_allowed_cmd_name.patch',
+    'GROMACS-2021.5_fix_threads_gpu_Gmxapitests.patch',
+    'GROMACS-2021.3_skip_test_for_plumed.patch',
+]
+checksums = [
+    'eba63fe6106812f72711ef7f76447b12dd1ee6c81b3d8d4d0e3098cd9ea009b6',  # gromacs-2021.5.tar.gz
+    # GROMACS-2019_fix_omp_num_threads_and_google_test_death_style_in_tests.patch
+    '406f5edd204be812f095a6f07ebc2673c5f6ddf1b1c1428fd336a80b9c629275',
+    # GROMACS-2019_increase_test_timeout_for_GPU.patch
+    '0d16f53d428155197a0ed0b0974ce03422f199d7c463c4a9156a3b99e3c86234',
+    # GROMACS-2021_fix_gmxapi_gmx_allowed_cmd_name.patch
+    'b7ffb292ec362e033db1bedd340353f0644dbaae872127750f3dda1ac7e87d49',
+    # GROMACS-2021.5_fix_threads_gpu_Gmxapitests.patch
+    '7d39da0b431fbc9e94e857552246eb0ec892f26b13e519b3706785cc5e01a563',
+    # GROMACS-2021.3_skip_test_for_plumed.patch    
+    'c06d0a8c3e75b232224f46bc3b70e135c4c02ec4151712970f5edd85d890f018',
+]
+
+builddependencies = [
+    ('CMake', '3.22.1'),
+    ('scikit-build', '0.11.1'),
+    ('Doxygen', '1.9.1'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('networkx', '2.6.3'),
+    ('PLUMED', _plumedver),
+    ('CUDA', '11.4.1', '', True),
+    ('UCX-CUDA', '1.11.2', versionsuffix),
```

Comment:
```suggestion
    ('UCX-CUDA', '1.11.2', _cuda_suffix),
```

Reply:
Done. We should have thought of it when I done the changes above. Slipped my mind as well.